packages/app/src/app/pages/Sandbox/Editor/Workspace/OpenedTabs/index.tsx

```diff
@@ -81,4 +81,4 @@ const OpenedTabs = ({ store, signals }) => {
   );
 };
 
-export default inject('signals', 'store')(observer(OpenedTabs));
+export default OpenedTabs;
```

Comment:
```suggestion

```

We prefer to use named exports instead of defaults, so could you please batch and commit these suggestions and then search the client files for any references to this file and update the imports?